### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MrD-RC**
Altered the Linux installation instructions, from the perspective of a Linux n00b.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,12 +41,40 @@ Depending on target operating system, _INAV Configurator_ is distributed as _sta
...
 
 1. Visit [release page](https://github.com/iNavFlight/inav-configurator/releases)
-1. Download Configurator for Linux platform (linux32 and linux64 are present)
-1. Extract tar.gz archive
-1. Make the following files executable:
-   * inav-configurator `chmod +x inav-configurator`
-   * (5.0.0+) chrome_crashpad_handler `chmod +x chrome_crashpad_handler`
-1. Run INAV Configurator app from unpacked folder
+2. Download Configurator for Linux platform (linux32 and linux64 are present)
+   *  **.rpm** is the Fedora installation file. Just download and run to install INAV Configurator
```
Comment (stronnag) [inline README.md:45]: This is not an executable file, you don't "run" it. You'd either:
 * In a terminal `sudo dnf localinstall FILE.rpm`  ; or
 * Open it with a package manager (e..g via the File Manager)

File: README.md [hunk 2]
```diff
@@ -41,12 +41,40 @@ Depending on target operating system, _INAV Configurator_ is distributed as _sta
...
+   *  **.deb** is the Debian/Ubuntu installation file. Just download and run to install INAV Configurator
```
Comment (stronnag) [inline README.md:46]: This is not an executable file, you don't "run" it. You'd either:
 * In a terminal `sudo apt install /path/to/FILE.deb`  ; or
 * Open it with a package manager (e.g. via the File Manager)

File: README.md [hunk 3]
```diff
@@ -41,12 +41,40 @@ Depending on target operating system, _INAV Configurator_ is distributed as _sta
...
+   *  **.tar.gz** is a universal archive. Download and continue with these instructions to install
+3. Change to the directory containing the downloaded **tar.gz** file
+4. download [this](https://raw.githubusercontent.com/iNavFlight/inav-configurator/master/assets/linux/inav-configurator.desktop) file to the same directory. Its filename should be `inav-configurator.desktop`.
+5. Extract **tar.gz** archive 
+```
+tar -C /tmp/ -xf INAV-Configurator_linuxNN_x.y.z.tar.gz
+```
+   **NN** is the bits of your OS. **x.y.z** is the INAV Configurator version number.
+
+6. If this is the first time installing INAV Configurator, create a home for it's files
```
Comment (stronnag) [inline README.md:56]: it's is a contraction (of it is), not a possessive, the apostrophe is not required.

**Comment by MrD-RC** [comment]
Thanks @stronnag 👍🏻

**Comment by stronnag** [comment]
> Thanks @stronnag 👍🏻

Apologies that it got split up into small pieces.

**Comment by MrD-RC** [comment]
No problem. It was easy to go through and update like that. It's all good :)

File: README.md (hunk 2 above)
Comment (mmosca) [reply README.md]: I am getting old, my suggestion would have been dpkg -i file.deb :)

File: README.md [hunk 4]
```diff
@@ -41,28 +41,60 @@ Depending on target operating system, _INAV Configurator_ is distributed as _sta
...
 
 1. Visit [release page](https://github.com/iNavFlight/inav-configurator/releases)
-1. Download Configurator for Linux platform (linux32 and linux64 are present)
-1. Extract tar.gz archive
-1. Make the following files executable:
-   * inav-configurator `chmod +x inav-configurator`
-   * (5.0.0+) chrome_crashpad_handler `chmod +x chrome_crashpad_handler`
-1. Run INAV Configurator app from unpacked folder
+2. Download Configurator for Linux platform (linux32 and linux64 are present)
+   *  **.rpm** is the Fedora installation file. Just download and install using `sudo dnf localinstall /path/to/INAV-Configurator_linuxNN_x.y.z..rpm` or open it with a package manager (e.g. via Files)
```
Comment (mmosca) [inline README.md:45]: Looks like we have 1 . too many in the rpm file name

File: README.md [hunk 5]
```diff
@@ -41,28 +41,60 @@ Depending on target operating system, _INAV Configurator_ is distributed as _sta
...
+sudo cp /opt/inav/inav-configurator/images/inav_icon_128.png /opt/inav/inav-configurator/icon
+```
+9. As a one off, move the desktop file into the applications directory 
+```
+sudo mv inav-configurator.desktop /usr/share/applications/
+```
+10. Make the following files executable:
+   * inav-configurator `chmod +x /opt/inav/inav-configurator/inav-configurator`
+   * (5.0.0+) chrome_crashpad_handler `chmod +x /opt/inav/inav-configurator/chrome_crashpad_handler`
+11. Run INAV Configurator app from unpacked folder
```
Comment (mmosca) [inline README.md:77]: From a beginner point of view, should we be obvious here and list the file path `/opt/inav/inav-configurator/inav-configurator`
or how to find it in applications?

File: README.md [hunk 6]
```diff
@@ -41,28 +41,60 @@ Depending on target operating system, _INAV Configurator_ is distributed as _sta
...
+
+#### Notes
 
 On some Linux distros, you may be missing `libatomic`, a `NW.JS` (specially `libnode.so`) dependency. If so, please install `libatomic` using your distro's package manager, e.g:
 
 * Arch Linux: `sudo pacman -S --needed libatomic_ops`
 * Debian / Ubuntu: `sudo apt install libatomic1`
 * Fedora: `sudo dnf install libatomic`
 
+1. Dont forget to add your user into dialout group "sudo usermod -aG dialout YOUR_USERNAME" for serial access
```
Comment (mmosca) [inline README.md:87]: Is dialout the correct group in all distros?

Should we teach the user how to figure out the correct group by listing the contents of /dev?

File: README.md [hunk 7]
```diff
@@ -41,28 +41,60 @@ Depending on target operating system, _INAV Configurator_ is distributed as _sta
...
 ### Mac
 
 1. Visit [release page](https://github.com/iNavFlight/inav-configurator/releases)
 1. Download Configurator for Mac platform
 1. Extract ZIP archive
 1. Run INAV Configurator
-1. Configurator is not signed, so you have to allow Mac to run untrusted application. There might be a monit for it during first run
 
-## Building and running INAV Configurator locally (for development or Linux users)
+## Building and running INAV Configurator locally (for development)
```
Comment (mmosca) [inline README.md:97]: or fun? :)

File: README.md (hunk 6 above)
Comment (stronnag) [reply README.md:87]: In Arch, it would be `uucp` ... but you'd expect an Archer to know that. `dialout` covers most cases.

File: README.md (hunk 5 above)
Comment (stronnag) [reply README.md:77]: Finding it in "Applications" depends on the user's selected DE. It will be different in GNOME, KDE, Xfce and the myriad of others.

Comment (mmosca) [reply README.md:77]: I usually would run things from the terminal anyway.

It is probably a good idea to make it obvious that you should run `/opt/inav/inav-configurator/inav-configurator` to launch inav as the readon why they did the chmod +x on the steop before may not be obvious to a beginner.

File: README.md (hunk 4 above)
Comment (MrD-RC) [reply README.md:45]: The filename wasn't quite right either. Corrected it and the .deb.

**Pull request merged**